README.md

```diff
@@ -4,7 +4,14 @@ This project represents a challenge to test your git flow, programming, and trou
 ## Challenge Instructions
 You will begin by creating a fork of this repository to your own Github account. All work will be completed and reviewed within your own fork. Please do not create a Pull Request (PR) back into this repository.
 
-Once you have created your fork, review the Issues labeled `challenge task` [HERE](https://github.com/Shift3/react-challenge-project/issues?q=is%3Aissue+is%3Aopen+label%3A%22challenge+task%22). Work on each task in a new branch. Once the work is completed, create a Pull Request from the working branch to your `master` branch and reference the Issue you are working on. These open PRs will be the final submission which Shift3 will review.
+Once you have created your fork, review the Issues labeled `challenge task` [HERE](https://github.com/Shift3/react-challenge-project/issues?q=is%3Aissue+is%3Aopen+label%3A%22challenge+task%22). Shift3's standard for branching is as follows:
+
+- Maintain a `master` and development (`dev`) branch on your fork
+- For each new feature (issue), create a branch based off of your `dev` branch
+    - Once your feature has been completed, submit a Pull Request from your feature branch to your `dev` branch
+    - If you have a reviewer on your repository, once your Pull Request has been approved, merge your feature branch back into `dev`
```

Comment:
I guess we can't stop them from having others review the code, hopefully people are not requesting help with any challenge 😅

Reply:
I added that there just in case we have proctors for future exams of this kind.